selenium/exercise-10.rb

```diff
@@ -0,0 +1,19 @@
+require 'selenium-webdriver'
+
+Selenium::WebDriver::Chrome::Service.driver_path = "C:/Users/anup.desai/Desktop/infuse/Practice-ANUPDESAI/selenium/geckodriver.exe"
+
+driver = Selenium::WebDriver.for :firefox
+
+driver.get "https://computer-database.gatling.io/computers"
+
+# element = driver.find_element(:tag_name, "tr")
+element = driver.find_element(:tag_name, "tbody")
+element = element.find_elements(:tag_name, "a")
```

Comment:
`a` is not a good locator strategy. Use `tr` tag to get all the row elements.
You then need to iterate over the each of the rows and read the desired column